### ISSUE TITLE
Unable to fetch My Business Reviews from a location.

### ISSUE THREAD

**Issue by ktist** [googleapis/google-api-nodejs-client#3513]
Since the `mybusiness.googleapis.com/v4` API is not managed (according to my knowledge and research at least) by the google-api-nodejs-client, and having the need to work with review data on my application both to fetch all locations' reviews and reply to some of them, I have resorted to manually invoke the the `client.request({url})` to perform this request.

To my understanding as of today, all of the `mybusiness.googleapis.com/v4` API is being deprecated, expect some features like review handling as per Google's documentation ([Review handling](https://developers.google.com/my-business/content/review-data) , [accounts.locations.reviews.list](https://developers.google.com/my-business/reference/rest/v4/accounts.locations.reviews/list) ) hence the reason why methods for calls to `mybusiness.googleapis.com/v4` API is not present on your library.

The `client.request({url})` works as intended for every other API URLs I need, so I can assume nothing is wrong with my codebase, although here is a brief example of the implementation: 
```
  const client = new OAuth2Client(CLIENT_ID, CLIENT_SECRET, REDIRECT_URI);
  client.setCredentials(tokens);
  const url = `https://mybusiness.googleapis.com/v4/accounts/-/locations/${loccationID}/reviews`;
  client
    .request({ url })
    .then((data) => {
      console.log(data);
    })
    .catch((err) => {
      console.log(err);
    });

```

No matter any different methods I have tried (even on google's oAuth playground), I am getting this error:
```
Sending a request to `https://mybusiness.googleapis.com/v4/accounts/accID/locations/locID/reviews`; returns the following message: 'Google My Business API has not been used in project "projectNumber" before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/mybusiness.googleapis.com/overview?project="projectNumber" then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.'
```

When I access the above URL I am getting a 404 error (as you will too if you check that by changing projectNumber to your project number) indicating that this API cannot really be enabled. When I search specifically  for the mybusinessapi, I can only find the 8 other APIs (like mybusinessnotifications etc) that, for troubleshooting purposes, have all been enabled.

I am missing a feature of the google-api-nodejs-client, or am I not reading the documentations ([Review handling](https://developers.google.com/my-business/content/review-data) , [accounts.locations.reviews.list](https://developers.google.com/my-business/reference/rest/v4/accounts.locations.reviews/list) ) correctly?

**Comment by julipopo** [comment]
Cause I've the same code than you and it works for me : 
Did you request access to the API "Google My Business API" ? last part of this guide : [https://developers.google.com/my-business/content/prereqs](https://developers.google.com/my-business/content/prereqs)

Once you've have been granted, you should see 9 results of "my business" by searching here (selecting your project) : [https://console.cloud.google.com/apis/library/browse](https://console.cloud.google.com/apis/library/browse) versus 8 results if you've not been granted.
Then don't forget to enable the new one, the one called "Google My Business API"

**Comment by ktist** [comment]
Hey @julipopo, thanks for your reply. 

I found the problem and I forgot to update this issue. So basically evn though we have specified in our Google My Business API access request form every email that needed to have access they simply forgot (most likely) to add the other programmer's and mine, hence why I could not view the proper API.

Anyways again thanks for taking the time to reply to me, I'll go ahead and close this issue.
